client/src/pages/Error.tsx

```diff
@@ -0,0 +1,5 @@
+const Error = (): JSX.Element => {
```

Comment:
this page is suggesting to be a component, as Not Found page, i think

```suggestion
const Error = (): React.FC => {
```

Reply:
Also, we don't need this page for now. Let's do it later